nemo/llama3-70b/llama3-70b.yaml

```diff
@@ -88,10 +87,17 @@ model:
   apply_rope_fusion: true
   attention_type: multihead
   share_embeddings_and_output_weights: false
+#  tokenizer:
+#    library: huggingface
+#    type: meta-llama/Meta-Llama-3-70B
+#    use_fast: true
   tokenizer:
-    library: huggingface
-    type: meta-llama/Meta-Llama-3-70B
-    use_fast: true
+    library: 'megatron'
+    type: 'GPT2BPETokenizer'
```

Comment:
don't have to do this now, but for fine-tuning or continued pretraining, we may want to use the correct tokenizer.

Reply:
sounds great -